packages/shopify-codemod/test/fixtures/convert-default-export-objects-to-named-exports/literals.input.js

```diff
@@ -0,0 +1,9 @@
+export default {
```

Comment:
```js
export default {
  foo() { return this.bar; }
}
```

Also, would be nice to export any functions as function declarations rather than expressions.